src/go/k8s/tests/e2e/regressions/00-assert.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: cluster
+status:
+  readyReplicas: 1
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+collectors:
+- type: pod
+  selector: app.kubernetes.io/name=redpanda
+  tail: -1
```

Comment:
This is not complete assertion. 

The best example is 
https://github.com/redpanda-data/redpanda/blob/60bd6df7ade970abb254949090cdfbb5d5db953f/src/go/k8s/tests/e2e/update-image-and-node-port/00-assert.yaml#L48-L60